src/generators/javascript/templates/index.hbs

```diff
@@ -18,13 +18,52 @@
 import Ajv, { ErrorObject } from 'ajv'
 import AjvDraft4 from 'ajv-draft-04'
 {{/if}}
-import * as Rudder from './rudder'
+
 {{#if isBrowser}}
+
+import type { RudderAnalytics, RudderAnalyticsPreloader, ApiOptions } from '@rudderstack/analytics-js';
+/**
+ * The analytics.js snippet should be available via window.analytics.
+ * You can install it by following instructions at: https://docs.rudderstack.com/stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk#2-installing-the-rudderstack-javascript-sdk
+ */
+declare global {
+  interface Window {
+    rudderanalytics: RudderAnalytics | RudderAnalyticsPreloader | undefined;
+  }
+}
+
 type apiCallback = (data?: any) => void;
+
+export interface Options extends ApiOptions {
```

Comment:
We need this for backward compatibility.

Reply:
As we discussed earlier, the `ApiOptions` type is a super type that should encompass `Options` and the `context` property inside it. We should replace `Options` with `ApiOptions`.

What breaking change were you observing otherwise?

Reply:
Addressed